ml-agents/mlagents/trainers/policy/checkpoint_manager.py

```diff
@@ -0,0 +1,95 @@
+# # Unity ML-Agents Toolkit
+from typing import Dict, Any, Optional, List
+import os
+import attr
+from mlagents.trainers.training_status import GlobalTrainingStatus, StatusType
+from mlagents_envs.logging_util import get_logger
+
+logger = get_logger(__name__)
+
+
+@attr.s(auto_attribs=True)
+class Checkpoint:
+    steps: int
+    file_path: str
```

Comment:
this file_path is the nn file path, I think in addition to the nn_file_path, this should also contain the tf checkpoint file path (Because we can only resume training from a tf checkpoint, not a nn)

Reply:
That should be managed automatically by the Tensorflow resume logic though, right?  It feels a little odd to couple this code with Tensorflow.

Reply:
I guess I just have a very limited understanding of what the CheckpointManager is doing (and I think it shows in a lot of the comments I made). Ignore this one.